src/main/resources/config.properties

```diff
@@ -21,4 +22,11 @@ build.workspace.computer.not.found = Unable to access the computer for this buil
 matlab.command.build.step.name = runMATLABCommand
 matlab.tests.build.step.name = runMATLABTests
 matlab.command.step.display.name = Run MATLAB commands, scripts, or functions
-matlab.tests.step.display.name = Run MATLAB tests and generate artifacts
\ No newline at end of file
+matlab.tests.step.display.name = Run MATLAB tests and generate artifacts
+matlab.custom.location = Custom...
+matlab.added.to.path.from = Prepending MATLAB to the PATH variable from this location: %s
+Axis.matlab.key = MATLAB
+Axis.use.matlab.warning = This project specifies MATLAB versions using the added 'MATLAB' axis as well as the 'Use MATLAB version' option. The value specified by 'Use MATLAB version' take precedence over the values specified by the 'MATLAB' axis.
+Axis.no.installed.matlab.error = Because no MATLAB versions are configured in Jenkins Global Tool Configuration, the plugin will run the default matrix configuration.
+Use.matlab.version.axis.warning = This project specifies MATLAB version using the 'Use MATLAB version' option as well as the added 'MATLAB' axis. The value specified by 'Use MATLAB version' take precedence over the values specified by the 'MATLAB' axis.
```

Comment:
1) Here in the first sentence version should be changed to versions. This is because multiple versions are specified using two options. 
2) In the second sentence, take should be takes.

Reply:
Done.